ecoaliface/simple.py

```diff
@@ -304,7 +304,23 @@ def get_feeder_max_run_time(self):
         self.log.debug("Received fuel status vals: %r", status_vals)
 
         return (status_vals[9] << 8 | status_vals[8]);
-        
+
+    def get_feeder_coal_left_percent(self):
+        _status = self.get_cached_status()
+        coal_left_percent = 100 - (
+            min(
+                round(
+                    _status.feeder_current_run_time
+                    / 60
```

Comment:
Aren't both feeder_current_run_time and feeder_max_run_time in seconds?

Reply:
Nope. Feeder current time is in seconds probably because it is calculated based on the amount of seconds the feeder has been running. Feeder max time is a setting specified in minutes by the user in the controller's settings.